.coderabbit.yaml

```
@@ -0,0 +1,84 @@
+      util_dirs: [ ]
+      essential_rules: true
+chat:
+  auto_reply: true
+knowledge_base:
+  opt_out: false
+  learnings:
+    scope: auto
+  issues:
+    scope: auto
+  jira:
+    project_keys: [ ]
```

yamllint

[error] too many spaces inside empty brackets

(brackets)